docs/fields/field-details.asciidoc

```diff
@@ -6434,6 +6469,22 @@ example: `test-pod-cdcws`
 
 // ===============================================================
 
+|
+[[field-orchestrator-resource-parent-type]]
+<<field-orchestrator-resource-parent-type, orchestrator.resource.parent.type>>
+
+| Type or kind of the parent resource of the resource being acted upon.
+
+type: keyword
+
+
+
+example: `DaemonSet`
+
```

Comment:
ideally we would enumerate the values we currently expect along with whether they are case-sensitive etc. This will help sensor writers. Perhaps we could reference a Kube docs page for the list of 'nouns' and standard spelling & capitalization.